dali/python/backend_impl.cc

```diff
@@ -284,8 +284,6 @@ void ExposeTensor(py::module &m) {
       Check if provided python object represent a valid DLPack capsule.
       It returns a tuple of two boolean values: one indicating if this is a valid DLPack object, and the other if the data
 
-      Parameters
-      ----------
```

Comment:
I don't see any difference in the resulting docs, what happened here?

Reply:
That is good. Apparently it was not needed, and only sphinx was complaining with:
```
docstring of nvidia.dali.backend.TensorListCPU.__init__:9: WARNING: Unexpected section title.
 Parameters
 ----------
```